src/magicbox/MagicBoxClear.ts

```diff
@@ -0,0 +1,26 @@
+import { $$, Dom } from '../utils/Dom';
+import { MagicBoxInstance } from './MagicBox';
+import { l } from '../strings/Strings';
+import { AccessibleButton } from '../utils/AccessibleButton';
+
+export class MagicBoxClear {
```

Comment:
Would it have been a good idea to use a utility function instead of a class?

Reply:
Could have been, it's kind of a personal preference.

I personally prefer to have a dedicated class as it's a logical/easy to understand piece of the component which can be pretty easily extracted. 

It's also easier to test like this, versus a a private utility function inside the original file.